storageserver/src/tests/testhelper.cpp

```diff
@@ -41,10 +41,8 @@ vdstestlib::DirConfig getStandardConfig(bool storagenode) {
     config = &dc.addConfig("stor-communicationmanager");
     config->set("rpcport", "0");
     config->set("mbusport", "0");
-    config = &dc.addConfig("stor-bucketdb");
     config->set("chunklevel", "0");
```

Comment:
Stale `stor-bucketdb` config can be removed here as well

Reply:
Thanks, fixed